build.sbt

```diff
@@ -1,12 +1,12 @@
 import com.typesafe.sbt.packager.docker._
 
-lazy val circeVersion        = "0.9.0"
-lazy val attoVersion         = "0.6.1"
+lazy val circeVersion        = "0.9.1"
+lazy val attoVersion         = "0.6.2-M1"
 lazy val catsEffectVersion   = "0.8"
 lazy val catsVersion         = "1.0.1"
-lazy val declineVersion      = "0.4.0"
+lazy val declineVersion      = "0.4.1"
```

Comment:
What's decline?

Reply:
It's an applicative command line parsing library that I'm using for the deployment stuff. It handles parsing commands options and printing help messages. The telnet server also uses it. A benefit beyond parsing is that every option is self-documenting so the help messages never get out of date.